rkt/list.go

```diff
@@ -207,3 +210,22 @@ func fmtNets(nis []netinfo.NetInfo) string {
 	}
 	return strings.Join(parts, ", ")
 }
+
+func getImageName(p *pod, appName types.ACName) (string, error) {
+	aim, err := p.getAppsImageManifests()
```

Comment:
I was trying to reuse this function in https://github.com/coreos/rkt/pull/1936 
Turns out getImageName is in a loop, so that `getAppsImageManifests` is called multiple times, while it actually only needs to be called once.
@blixtra